src/helpers/index.tsx

```diff
@@ -1,5 +1,5 @@
-import { addresses, EPOCH_INTERVAL, BLOCK_RATE_SECONDS, BONDS } from "../constants";
```

Comment:
This file shouldn't be a `.tsx` and be `ts` instead.

Reply:
hey, I can change this back to .ts, but functions like `getPairImage`, `getEthTokenImage` will need to be moved out into a separate file, lmk if you want me to change back to ts and move functions to separate file or keep as tsx

Reply:
why do those have to be moved out? i dont see the dependency.

Reply:
Because .ts files can't hold jsx/tsx components inside of them. At least the way our tsconfig is currently set up, it complains.